Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.xaml

```diff
@@ -9,18 +9,29 @@
             <SolidColorBrush x:Key="MarkdownBackgroundBrush" Color="#FFF6F8FA" />
             <SolidColorBrush x:Key="MarkdownInlineCodeBackgroundBrush" Color="#FFDDDFE1" />
             <SolidColorBrush x:Key="MarkdownQuoteForeground" Color="#FF6E747C" />
+            <SolidColorBrush x:Key="MarkdownRichTextBlockBackground" Color="#FFF6F8FA" />
         </ResourceDictionary>
         <ResourceDictionary x:Key="Dark">
             <SolidColorBrush x:Key="MarkdownBorderBrush" Color="#FF424242" />
             <SolidColorBrush x:Key="MarkdownBackgroundBrush" Color="#FF202020" />
             <SolidColorBrush x:Key="MarkdownInlineCodeBackgroundBrush" Color="#FF363636" />
             <SolidColorBrush x:Key="MarkdownQuoteForeground" Color="#FFBEC1C5" />
+            <SolidColorBrush x:Key="MarkdownRichTextBlockBackground" Color="#FF202020" />
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="HighContrast">
+            <SolidColorBrush x:Key="MarkdownBorderBrush" Color="#FF424242" />
+            <SolidColorBrush x:Key="MarkdownBackgroundBrush" Color="#FF202020" />
+            <SolidColorBrush x:Key="MarkdownInlineCodeBackgroundBrush" Color="#FF363636" />
+            <SolidColorBrush x:Key="MarkdownQuoteForeground" Color="#FFBEC1C5" />
+            <SolidColorBrush x:Key="MarkdownRichTextBlockBackground" Color="{ThemeResource SystemColorWindowColor}" />
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
+   
     <Style TargetType="controls:MarkdownTextBlock">
+        <Setter Property="Background" Value="{ThemeResource MarkdownRichTextBlockBackground}" />
         <Setter Property="IsTabStop" Value="False" />
-        <Setter Property="LinkForeground" Value="{ThemeResource SystemControlHighlightAccentBrush}" />
+        <Setter Property="LinkForeground" Value="{ThemeResource SystemColorHotlightColor}" />
```

Comment:
I had a problem adding this option. The SystemColorHotlightcolor doesn't exist in the default common control colors list. Which compilation version are you using?

Reply:
Sorry - I replied on email but it didn't seem to attach the answer to the right post.

This is defined in “C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\DesignTools\SystemThemes\Uwp\SystemColors.xaml”. Documented here https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-theme-resources - I was following the guidance here https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/high-contrast-themes

TonyG

Reply:
I tested this on 17763 and 17134 and it works as expected. @lucaasrojas, what build are you running into an issue?

Reply:
I tried it again and it worked. Sorry for the mistake.